src/pages/projects/CreateProject.tsx

```diff
@@ -135,6 +142,7 @@ const CreateProject: FC = () => {
         .then(async () => {
           if (
             !values.default_instance_storage_pool ||
+            !values.default_project_network ||
             values.features_profiles === false
```

Comment:
This condition is not correct. We should ensure both -- storage and network -- are unset, not one or the other for the early exit.
```suggestion
            (!values.default_instance_storage_pool &&
            !values.default_project_network) ||
            values.features_profiles === false
```

Reply:
Best adjust the logic below to cater for the case that network can be "none" and avoid setting it in the payload in that case.

Reply:
Ah, that's a great solution to what we were discussing earlier, thank you!